game-core/src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -495,9 +494,9 @@ private Element getSingleChild(final String name, final Node node, final boolean
         .collect(Collectors.toList());
   }
 
-  private static List<Node> getNonTextNodesIgnoring(final Node node, final String ignore) {
+  private static List<Node> getNonTextNodesIgnoring(final Node node) {
```

Comment:
I would've renamed to method to `getNonTextNodesIgnoringValue` or something

Reply:
Updated the method name per suggestion: 95f4abd8d